### PR TITLE
fix many broken links

### PR DESCRIPTION
**Pull request by redallen**
There's still some broken, I'm sure.

### CODE REVIEW

File: packages/v4/src/content/design-guidelines/components/login-page/login-page.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ section: components
 
 The **login page** allows a user to gain access to an application by entering their username and password or by authenticating using a social media login. Related design guideline: [Login](/components/login-page/design-guidelines)
 
+The login page allows a user to gain access to an application by entering their username and password or by authenticating using a social media login.
```
Comment (mmenestr) [inline packages/v4/src/content/design-guidelines/components/login-page/login-page.md:8]: This line shouldn't have been added. Now it repeats twice (I think it overrides her changes)

File: packages/v4/src/content/design-guidelines/components/text-input/text-input.md [hunk 2]
```diff
@@ -16,29 +16,29 @@ Use text input to gather free-form text from a user.
...
-* Provide a [text area](/documentation/react/components/textarea) for input anticipated to be longer than a single line.
+* When a value is too long to be fully displayed in a text field, use [truncation](/ux-writing/truncation).
+* Provide a [text area](/components/text-area) for input anticipated to be longer than a single line.
 * Pre-populate fields with known values when possible, like a default IP address, for example.
 
 ### Help
 **Placeholder text**
 
-[Placeholder text](/design-guidelines/usage-and-behavior/forms#placeholder-text) is text inside a form field that provides an example of the required or recommended format for text input. For example, placeholder text for a date might be "DD/MM/YYYY".
+[Placeholder text](/components/forms/design-guidelines#placeholder-text) is text inside a form field that provides an example of the required or recommended format for text input. For example, placeholder text for a date might be "DD/MM/YYYY".
```
Comment (mmenestr) [inline packages/v4/src/content/design-guidelines/components/text-input/text-input.md:26]: The link should be /components/form/design-guidelines.... etc,  with "form" being singular, not plural

File: packages/v4/src/content/design-guidelines/components/text-input/text-input.md [hunk 3]
```diff
@@ -16,29 +16,29 @@ Use text input to gather free-form text from a user.
...
 
 **Helper text**
 
-[Helper text](/design-guidelines/usage-and-behavior/forms#helper-text) is text below a form field that helps a user provide the right information, like "Enter a unique name".
+[Helper text](/components/forms/design-guidelines#helper-text) is text below a form field that helps a user provide the right information, like "Enter a unique name".
```
Comment (mmenestr) [inline packages/v4/src/content/design-guidelines/components/text-input/text-input.md:30]: Same here: The link should be /components/form/design-guidelines.... etc,  with "form" being singular, not plural

File: packages/v4/src/content/design-guidelines/components/text-input/text-input.md [hunk 4]
```diff
@@ -16,29 +16,29 @@ Use text input to gather free-form text from a user.
...
 
 
 **Popovers**
 
-[Popovers](/design-guidelines/usage-and-behavior/forms#popovers) provide context around a text or input field. Popovers are represented with an unfilled question mark con (fa-question-circle) and displayed when the user clicks on the icon.
+[Popovers](/components/forms/design-guidelines#popovers) provide context around a text or input field. Popovers are represented with an unfilled question mark con (fa-question-circle) and displayed when the user clicks on the icon.
```
Comment (mmenestr) [inline packages/v4/src/content/design-guidelines/components/text-input/text-input.md:35]: Same here: The link should be /components/form/design-guidelines.... etc,  with "form" being singular, not plural

File: packages/v4/src/content/design-guidelines/components/text-input/text-input.md [hunk 5]
```diff
@@ -16,29 +16,29 @@ Use text input to gather free-form text from a user.
...
 
-For more information about how to provide help content, see  the [forms design guidelines](/design-guidelines/usage-and-behavior/forms#user-help).
+For more information about how to provide help content, see  the [forms design guidelines](/components/forms/design-guidelines#user-help).
```
Comment (mmenestr) [inline packages/v4/src/content/design-guidelines/components/text-input/text-input.md:37]: Same here: The link should be /components/form/design-guidelines.... etc,  with "form" being singular, not plural

File: packages/v4/src/content/design-guidelines/components/text-input/text-input.md [hunk 6]
```diff
@@ -47,5 +47,5 @@ Wherever possible, use inline errors to notify users where there is an issue wit
 
 <img src="./img/error.png" alt="error and validation example" width="320"/>
 
-For more information about how to provide errors and validation, see  the [forms design guidelines](/design-guidelines/usage-and-behavior/forms#errors-and-validation).
+For more information about how to provide errors and validation, see  the [forms design guidelines](/components/forms/design-guidelines#errors-and-validation).
```
Comment (mmenestr) [inline packages/v4/src/content/design-guidelines/components/text-input/text-input.md:50]: Same here: The link should be /components/form/design-guidelines.... etc,  with "form" being singular, not plural

**Review by mmenestr — CHANGES_REQUESTED** [review]
Just a few form links that need fixing + one override on Becca's work - need to delete the extra line!
Thanks!

File: packages/v4/src/content/design-guidelines/components/charts/charts.md [hunk 7]
```diff
@@ -151,7 +151,7 @@ For donut charts, don’t try to represent more than six categories. We recommen
 4. **Chart tooltip:** Chart tooltips appear on hover for both threshold segments and utilization segments. We recommend stating the threshold name, the segment name, and the value being represented. For example, if the threshold being met is "Danger," and the segment represents "Storage capacity," and the value being represented is 92%, your chart tooltip would state, "Danger: Storage capacity: 92%." We recommend stating the threshold name "at" the percentage the threshold begins (ex. Warning at 60%).
 
 #### React component
-[Donut utilization](/documentation/react/charts/chartdonututilization/#donut-utilization-threshold-examples)
+[Donut utilization](/charts/donut-utilization-chart/#donut-utilization-threshold-examples)
```
Comment (beccaglass) [inline packages/v4/src/content/design-guidelines/components/charts/charts.md:154]: should be /charts/donut-utilization-chart/react#donut-utilization-threshold-examples

File: packages/v4/src/content/design-guidelines/components/text-input/text-input.md (hunk 3 above)
Comment (beccaglass) [inline packages/v4/src/content/design-guidelines/components/text-input/text-input.md]: should be /components/form/design-guidelines#helper-text 

changed forms > form

File: packages/v4/src/content/design-guidelines/components/text-input/text-input.md (hunk 4 above)
Comment (beccaglass) [inline packages/v4/src/content/design-guidelines/components/text-input/text-input.md]: change to /components/popover/design-guidelines

File: packages/v4/src/content/design-guidelines/components/text-input/text-input.md (hunk 5 above)
Comment (beccaglass) [inline packages/v4/src/content/design-guidelines/components/text-input/text-input.md]: /components/form/design-guidelines#user-help

File: packages/v4/src/content/design-guidelines/components/text-input/text-input.md (hunk 6 above)
Comment (beccaglass) [inline packages/v4/src/content/design-guidelines/components/text-input/text-input.md]: /components/form/design-guidelines#errors-and-validation

File: packages/v4/src/content/get-started/design.md [hunk 8]
```diff
@@ -105,41 +105,37 @@ PatternFly offers a variety of resources to help you make the most of the design
...
 What typography should you be using? What about colors? Spacing? Check out styles to learn everything you need to know about our foundational design elements.
 
-[**View styles**](/design-guidelines/styles/colors) <i class="ws-content-blueArrow fas fa-arrow-right pf-u-mx-sm"></i>
+[**View styles**](/guidelines/colors) <i class="ws-content-blueArrow fas fa-arrow-right pf-u-mx-sm"></i>
 
 ### Get familiar with usage and behavior guidelines.
 Should you use a wizard or a form to solve this problem? How do you write a good button label? Which navigation makes the most sense? Check out our design guidelines to choose and use the right components for your use case.
 
-[**Usage and behavior guidelines**](/design-guidelines/usage-and-behavior/about-modal) <i class="ws-content-blueArrow fas fa-arrow-right pf-u-mx-sm"></i>
+[**Usage and behavior guidelines**](/guidelines/about-modal/design-guidelines) <i class="ws-content-blueArrow fas fa-arrow-right pf-u-mx-sm"></i>
```
Comment (beccaglass) [inline packages/v4/src/content/get-started/design.md:113]: should be /components/about-modal/design-guidelines

File: packages/v4/src/pages/home.js [hunk 9]
```diff
@@ -122,7 +122,7 @@ const HomePage = () => (
                     aria-label="view components"
                     className="pf-u-pl-0 pf4-c-link__text ws-mdx-a"
                   >
-                    <Link to="/documentation/react/components/aboutmodal">
+                    <Link to="/components/aboutmodal">
```
Comment (beccaglass) [inline packages/v4/src/pages/home.js:125]: /components/about-modal

File: packages/v4/src/content/design-guidelines/components/data-list/data-list.md [hunk 10]
```diff
@@ -174,11 +174,11 @@ Use a compound expandable table when:
 * It would not make sense to combine all of this information into a single, simple expansion.
 
 ### Actionable table
-The actionable table provides checkboxes that enable users to select one or more rows and act on that selection using options in the [toolbar](/design-guidelines/usage-and-behavior/toolbar).
+The actionable table provides checkboxes that enable users to select one or more rows and act on that selection using options in the [toolbar](/components/toolbar/design-guidelines).
 
 <img src="./img/actionable-table.png"  alt="Actionable table"  width="1174"/> 
 
-1. **[Bulk selection](/design-guidelines/usage-and-behavior/bulk-selection):** When present, selects all items in a table. If pagination is being used, this will only select items on the current page. See [bulk selection](/design-guidelines/usage-and-behavior/bulk-selection) for more information. 
+1. **[Bulk selection](/guidelines/bulk-selection):** When present, selects all items in a table. If pagination is being used, this will only select items on the current page. See [bulk selection](/guidelines/bulk-selection) for more information.
```
Comment (beccaglass) [inline packages/v4/src/content/design-guidelines/components/data-list/data-list.md:181]: no bulk select guidelines

File: packages/v4/src/content/design-guidelines/components/data-list/data-list.md [hunk 11]
```diff
@@ -117,8 +117,8 @@ The elements mentioned below are similar for a table with compact or default spa
 
 <img src="./img/compact-data-table.png"  alt="Compact table"  width="1186"/> 
 
-1. **[Toolbar](/design-guidelines/usage-and-behavior/toolbar):** Sits above the table and contains controls for manipulating table data. Common actions include filtering, sorting, and pagination.
-2. **[Bulk selection](/design-guidelines/usage-and-behavior/bulk-selection):** When present, selects all items in a table. If pagination is being used, this will only select items on the current page. See [bulk selection](/design-guidelines/usage-and-behavior/bulk-selection) for more information.
+1. **[Toolbar](/components/toolbar/design-guidelines):** Sits above the table and contains controls for manipulating table data. Common actions include filtering, sorting, and pagination.
+2. **[Bulk selection](/guidelines/bulk-selection):** When present, selects all items in a table. If pagination is being used, this will only select items on the current page. See [bulk selection](/guidelines/bulk-selection) for more information.
```
Comment (beccaglass) [inline packages/v4/src/content/design-guidelines/components/data-list/data-list.md:121]: no bulk select guidelines

File: packages/v4/src/content/design-guidelines/components/data-list/data-list.md [hunk 12]
```diff
@@ -69,11 +69,11 @@ Every data list can be extended with these functionalities:
 * **[Expandable data list](#expandable-data-list)**
 
 ### Actionable data list
-The actionable data list provides checkboxes that enable users to select one or more rows and act on that selection using options in the [toolbar](/design-guidelines/usage-and-behavior/toolbar).
+The actionable data list provides checkboxes that enable users to select one or more rows and act on that selection using options in the [toolbar](/components/toolbar/design-guidelines).
 
 <img src="./img/actionable-list.png"  alt="Actionable list"  width="1172"/> 
 
-1. **[Bulk selection](/design-guidelines/usage-and-behavior/bulk-selection):** When present, selects all items in a table. If pagination is being used, this will only select items on the current page. See [bulk selection](/design-guidelines/usage-and-behavior/bulk-selection) for more information. 
+1. **[Bulk selection](/guidelines/bulk-selection):** When present, selects all items in a table. If pagination is being used, this will only select items on the current page. See [bulk selection](/guidelines/bulk-selection) for more information.
```
Comment (beccaglass) [inline packages/v4/src/content/design-guidelines/components/data-list/data-list.md:76]: no bulk select guidelines

File: packages/v4/src/content/design-guidelines/components/data-list/data-list.md [hunk 13]
```diff
@@ -42,8 +42,8 @@ The elements mentioned below are similar for a data list with compact or default
 
 <img src="./img/compact-list.png"  alt="Compact data list"  width="1188"/> 
 
-1. **[Toolbar](/design-guidelines/usage-and-behavior/toolbar):** The toolbar sits above the list and contains controls for manipulating list data. Common actions include filtering, sorting, and pagination.
-2. **[Bulk selection](/design-guidelines/usage-and-behavior/bulk-selection):** When present, selects all items in a table. If pagination is being used, this will only select items on the current page. See [bulk selection](/design-guidelines/usage-and-behavior/bulk-selection) for more information.
+1. **[Toolbar](/components/toolbar/design-guidelines):** The toolbar sits above the list and contains controls for manipulating list data. Common actions include filtering, sorting, and pagination.
+2. **[Bulk selection](/guidelines/bulk-selection):** When present, selects all items in a table. If pagination is being used, this will only select items on the current page. See [bulk selection](/guidelines/bulk-selection) for more information.
```
Comment (beccaglass) [inline packages/v4/src/content/design-guidelines/components/data-list/data-list.md:46]: I don't see bulk selection guidelines, so I think this link doesn't exist.

**Review by beccaglass — COMMENTED** [review]
Caught a couple errors... I believe Margot is reviewing as well so between us we should be able to catch most tweaks :)

File: packages/v4/src/content/design-guidelines/components/data-list/data-list.md (hunk 10 above)
Comment (redallen) [reply packages/v4/src/content/design-guidelines/components/data-list/data-list.md:181]: These are going to be added back soon. I'm fine with the link being broken for now.

**Review by mmenestr — APPROVED** [review]
Thanks!

**Pull request merged**